App.js

```diff
@@ -14,13 +14,13 @@ const App = () => {
   }, []);
 
   return (
-    <FlagsProvider flags={buildTimeFlags}>
+    <FlagsContext.Provider value={useState(buildTimeFlags)}>
```

Comment:
Can `useState` be moved into `flags.js` so that this is just the provider here? Now that we control the API, may as well locate it inside the flags.js because nothing else should care, own, create or touch the direct state value.

Perhaps `const { FlagsProvider } = createFlags()`

Reply:
are you alright with having to use a ```// eslint-disable-next-line react-hooks/rules-of-hooks``` in that declaration?